apps/spruce/src/pages/Version.tsx

```diff
@@ -32,11 +37,10 @@ export const VersionPage: React.FC = () => {
   const dispatchToast = useToastContext();
 
   // This query is used to fetch the version data.
-  const {
-    data: versionData,
-    error: versionError,
-    loading: versionLoading,
-  } = useQuery<VersionQuery, VersionQueryVariables>(VERSION, {
+  const { data: versionData, error: versionError } = useQuery<
+    VersionQuery,
+    VersionQueryVariables
+  >(VERSION, {
```

Comment:
Why don't we just poll this query instead of creating a separate query?

Reply:
I was thinking of doing that but remembered that some users were sensitive to amount of network traffic utilized and I thought it would be a good idea to keep it lower if possible. Here is a ticket that came from user feedback years ago: https://jira.mongodb.org/browse/EVG-12420

Reply:
I don't think this makes a difference you are just adding a second query that will be polling.

This will now run 2 queries for every page load and then poll 1 of them vs Just polling the original query.